ja: Make tutorials/hello-minikube.md follow v1.17 of the original text

ryokotmng opened the pull request:
Issue https://github.com/kubernetes/website/issues/21366

I referred [this doc](https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b) to fix.

### What I'm not pretty confident
I may have to change the 'Output is similar to:' wording to emphasize that users possibly see different output in some part?

### Appreciate
Since it's my very first PR for OSS, would be great if reviewers let me know any ideas and preferences where and how to fix or what to do if I missed something, even the smallest details. Thanks you vm for your review.

ryokotmng commented on content/ja/docs/tutorials/hello-minikube.md:
```diff
@@ -109,19 +110,19 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
 ## Serviceの作成
 
-通常、PodはKubernetesクラスタ内部のIPアドレスからのみアクセスすることができます。`hello-node`コンテナをKubernetesの仮想ネットワークの外部からアクセスするためには、Kubernetesの[*Service*](/ja/docs/concepts/services-networking/service/)としてポッドを公開する必要があります。
+通常、PodはKubernetesクラスタ内部のIPアドレスからのみアクセスすることができます。`hello-node`コンテナをKubernetesの仮想ネットワークの外部からアクセスするためには、Kubernetesの[*Service*](/ja/docs/concepts/services-networking/service/)としてPodを公開する必要があります。
```
changed 'ポッド' -> Pod

ryokotmng commented:
/assign @nasa9084

oke-py commented:
@ryokotmng Thank you for contribution!
It seems that the PR title and referenced issue number are wrong.
Could you edit the PR title and description?

Additionally, please comment `/assign` in the related issue to assign it to you.

ryokotmng commented:
Ahhhh, thank you for the advise. Will do quickly!

ryokotmng commented:
I think now the title and description are correct, and changed the commit message of the first commit of this PR as well. And I added myself to the issue. Thanks in advance for your review:)

oke-py commented:
@ryokotmng Thank you, but the description's 1st line is still wrong.
If you write only `#21366` instead of `https://github.com/kubernetes/website/issues/21366`, it will work.

ryokotmng commented:
Apologies for my mistake🙇‍♂️ Now fixed!

oke-py commented:
Good!

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
```diff
@@ -15,7 +15,7 @@ card:
 {{% capture overview %}}
 
-このチュートリアルでは、[Minikube](/docs/getting-started-guides/minikube)とKatacodaを使用して、Kubernetes上でシンプルなHello WorldのNode.jsアプリケーションを動かす方法を紹介します。Katacodaはブラウザで無償のKubernetes環境を提供します。
+このチュートリアルでは、[Minikube](/docs/setup/learning-environment/minikube)とKatacodaを使用して、Kubernetes上でシンプルなHello WorldのNode.jsアプリケーションを動かす方法を紹介します。Katacodaはブラウザで無償のKubernetes環境を提供します。
```
```suggestion
このチュートリアルでは、[Minikube](/ja/docs/setup/learning-environment/minikube)とKatacodaを使用して、Kubernetes上でシンプルなHello WorldのNode.jsアプリケーションを動かす方法を紹介します。Katacodaはブラウザで無償のKubernetes環境を提供します。
```
翻訳済みのためjaにリンクしておきます。

ryokotmng replied:
ありがとうございます🙇‍♂️

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -79,19 +79,20 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
-    出力:
+    出力は下記のようになります:
 
     ```shell
````
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L59

ryokotmng replied:
Apologies🙇‍♂️

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -79,19 +79,20 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
+
+    出力は下記のようになります:
 
     ```shell
````
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L74

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -130,7 +131,7 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
-    出力:
+    出力は下記のようになります:
 
     ```shell
````
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L106

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
 4. Katacoda環境のみ：ターミナル画面上部の+ボタンをクリックして **Select port to view on Host 1** をクリックしてください。
 
-5. Katacoda環境のみ：`30369`(Service出力に表示されている`8080`の反対側のポートを参照)を入力し、クリックしてください。
+5. Katacoda環境のみ：`8080`の反対側のService出力に、5桁のポート番号が表示されます。このポート番号はランダムに生成されるため、ここで使用するポート番号と異なる場合があります。ポート番号テキストボックスに番号を入力し、ポートの表示をクリックしてください。前の例の場合は、「30369」と入力します。
```
```suggestion
5. Katacoda環境のみ：`8080`の反対側のService出力に、5桁のポート番号が表示されます。このポート番号はランダムに生成されるため、ここで使用するポート番号と異なる場合があります。ポート番号テキストボックスに番号を入力し、ポートの表示をクリックしてください。前の例の場合は、`30369`と入力します。
```

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
-    出力:
+    出力は下記のようになります:
 
     ```shell
````
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L134

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
     ```
-   
+
 2. ここでは例として`heapster`のアドオンを有効化します:
````
```suggestion
2. ここでは例として`metrics-server`のアドオンを有効化します:
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L158-L160

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
 2. ここでは例として`heapster`のアドオンを有効化します:
 
     ```shell
     minikube addons enable heapster
````
```suggestion
    minikube addons enable metrics-server
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L163-L164

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
+
+    出力は下記のようになります:
 
     ```shell
````
```suggestion
    ```
```

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
+    出力は下記のようになります:
 
     ```shell
     heapster was successfully enabled
````
```suggestion
    metrics-server was successfully enabled
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L169-L175

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -205,31 +208,35 @@ Minikubeはビルトインのアドオンがあり、有効化、無効化、あ
 
     ```shell
````
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L185-L186

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -205,31 +208,35 @@ Minikubeはビルトインのアドオンがあり、有効化、無効化、あ
-    出力:
+    出力は下記のようになります:
 
     ```shell
````
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L222

oke-py commented on content/ja/docs/tutorials/hello-minikube.md:
````diff
@@ -205,31 +208,35 @@ Minikubeはビルトインのアドオンがあり、有効化、無効化、あ
 
     ```shell
-    heapster was successfully disabled
+    etrics-server was successfully disabled
````
```suggestion
    metrics-server was successfully disabled
```

ryokotmng commented:
Thanks for your comments! Will revise as you kindly suggested🙏 💦

ryokotmng commented:
@oke-py Revised and double-checked by myself🙏 you have very good eyes! Would be great if you could go through~:)

inductor commented:
/approve
@ryokotmng Thank you for your contribution :)
